simulation_parameters/Constants.cs

```diff
@@ -1826,6 +1827,24 @@ public static class Constants
     public const float TOLERANCE_MAXIMUM_SURVIVABLE_PRESSURE_DIFFERENCE = 4000000;
     public const float TOLERANCE_PERFECT_THRESHOLD_PRESSURE = 350000;
 
+    public const int TEMPERATURE_COLD_MIN = -4;
```

Comment:
I think these should be placed nearby to planet generator variables, because right now it doesn't make sense to insert these between `TOLERANCE_PERFECT_THRESHOLD_PRESSURE` and `TOLERANCE_INITIAL_PRESSURE_MIN_FRACTION` as those are two *very* closely linked constants to tolerance pressure specifically.